packages/contracts-bsc/contracts/BscTokenController.sol

```diff
@@ -70,16 +71,13 @@ contract BscTokenController {
 
     ITrueCurrency public token;
     IRegistry public registry;
-    address public fastPause; // deprecated
-    address public trueRewardManager; // deprecated
```

Comment:
Just like with #1242, let's separate {removal of deprecated storage variables} and {adding new features} into different PRs.

I would have been fine approving and merging the removal, but (re)adding functionality here has more scrutiny.

Reply:
will separate it to a new pr

Reply:
Thanks for splitting it!

However, I noticed that your new PR #1248 still has changes that come from this PR. Ideally PRs capture independent changes, so we can review and merge one PR without necessarily waiting for another PR to pass review. As of now, I'm unwilling to begin reviewing another PR that requires this one to be merged first, especially if there are unresolved comments in this PR whose fixes will need to be propagated.

Reply:
Thanks for your great advice.